extra/redisotel/tracing.go

```diff
@@ -41,6 +68,19 @@ func InstrumentTracing(rdb redis.UniversalClient, opts ...TracingOption) error {
 
 		rdb.OnNewNode(func(rdb *redis.Client) {
 			opt := rdb.Options()
+			host, portString, err := net.SplitHostPort(opt.Addr)
+			if err == nil {
+				opts = append(opts, WithAttributes(
+					semconv.ServerAddress(host),
+				))
+				// Parse the port string to an integer
+				port, err := strconv.Atoi(portString)
+				if err == nil {
+					opts = append(opts, WithAttributes(
+						semconv.ServerPort(port),
+					))
+				}
```

Comment:
Could you please add a helper function so we don't repeat this block 3 times?

Reply:
done :-) thanks!